File: fix_defer.js
const fs = require('fs');
const path = require('path');

const dir = __dirname;
const files = fs.readdirSync(dir).filter(f => f.endsWith('.html'));

let count = 0;
for (const file of files) {
    const filePath = path.join(dir, file);
    const content = fs.readFileSync(filePath, 'utf8');
    const modified = content.replace(/ defer><\/script>/g, '></script>');
    
    if (content !== modified) {
        fs.writeFileSync(filePath, modified, 'utf8');
        count++;
    }
}
console.log(`Fixed ${count} files.`);

File: fix_defer.py
import os
import glob

html_files = glob.glob('*.html')
count = 0
for file in html_files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    modified = content.replace(' defer></script>', '></script>')
    
    if content != modified:
        with open(file, 'w', encoding='utf-8') as f:
            f.write(modified)
        count += 1
print(f"Fixed {count} files.")
